tests/lib/__init__.py

```diff
@@ -895,50 +899,55 @@ def _git_commit(
 
 
 def _vcs_add(
-    script: PipTestEnvironment,
+    location: pathlib.Path,
     version_pkg_path: pathlib.Path,
     vcs: str = "git",
 ) -> pathlib.Path:
     if vcs == "git":
-        script.run("git", "init", cwd=version_pkg_path)
-        script.run("git", "add", ".", cwd=version_pkg_path)
-        _git_commit(script, version_pkg_path, message="initial version")
+        Git.run_command(["init"], cwd=os.fspath(version_pkg_path))
```

Comment:
I wonder if these should use subprocess directly instead, to avoid relying on the implementation we should be testing.